src/Integration.Vsix.UnitTests/packages.lock.json

```diff
@@ -29,18 +29,6 @@
           "Microsoft.CodeCoverage": "16.6.1"
         }
       },
-      "Microsoft.VisualStudio.Sdk.TestFramework": {
```

Comment:
Removing the reference to the TestFramework package has made a lot of changes to the `Integration.Vsix.UnitTests` `packages.lock.json` file. Fingers crossed the CI builds pass...